### ISSUE TITLE
WHY I WAS UNABLE TO HELP OUT?

### ISSUE THREAD

**Issue by ret394temp** [niryarden/sweetcase#4]
WHY I WAS UNABLE TO HELP OUT?  IS TITLE.

#  NOTE - ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.

#  I AM RET394. THOUGHT I FORGOT SO WHO WOULD REMEMBER.

# THIS IS TOO LONG AS I THINK I GET ONLY ONE SHOT. IN CASE YOU WEER THE ONE THAT BANNED ME. I HAVE NO IDEA, AND I DONT CARE. I WILL STILL ROAST SPAMMERS AND SCAMMERS AND SHITTY WEB FRAMEWORKS THAT MAKE ME CRY.

GITHUB -GET OUT 
GITHUB OR SOME USER BANNED BE AGAIN AND I WAS LIKE I AM DONE.

SO I NEED TO GET THAT ACCOUNT BACK. ANY TIPS?

I AM SO SORRY BUT I COULD NOT HELP OUT WITH OTHER PROJECTS AS WELL. AND OPEN SOURCE IS DYING BECASUE OF GITHUB. FOR MORE PROOF - WAIT TILL THEY SET ME FREE.

<H1> WHY I FEEL THIS NEW ACCOUNT IS A BAD IDEA  </H1>

## I GET CONFUSED WITH ALL THESE ACCOUNTS. I JUST WANTED ONE GITHUB ACCOUNT TO HOST CODE AND ROAST BAD PEOPLE POSTING CODE THAT KILLED PEOPLE IN COVID 19

# ME STORY LONG, PLEASE SOMEONE ANYONE, HELP ME I AM NOT EVIL. GITHUB IS.

# TLDR - I ALSO BLACKMAILED GITHUB WITHOUT ANY EVIDENCE WHEN I GREW DESPERATE. AS THEY WERE LIKE ZERO TRANSPARENCY.

# OF COURSE I WILL ONLY CONTRIBTUE TO NIRYARDEN'S PROJECTS AND ONLY THIS USER WILL MATTER ONCE I COMPLETE MY SECRET PLAN ON MY PATREON.

<H1> 

PATREON.COM/RET394

AND I GOT A  YOUTUBE FORGOT WHAT ITS LINK WAS

AND GOT A HATE BONER FOR PATREON, GITHUB, HACKERNOON, GITLAB.

ALL BLOCKING ME WITHOUT LETTING ME TALK. JUST LET ME TALK.
JUST LET ME TALK. JUST LET ME TALK.

HEY FOUND MY OWN YOUTUBE 
 https://www.youtube.com/channel/UCfh8ljK0lvvNWT6njGzweew/videos
ENGAGIN CONTENT  https://www.youtube.com/channel/UCfh8ljK0lvvNWT6njGzweew/discussion

ENGAGIN CONTENT   IS HIDDEN THERE. WEIRD CAN YOUTUBE DELETE COMMENTS LEAKING MY REAL NAME, SOHRTENED TO MC.

I THOUGHT USING RON TLAIB AS AN ALIAS WOULD PROTECT ME FROM THESE GUYS, BUT STILL GOT BURNT. SO ALL MY ACCOUNTS NOW USE MY REAL NAME - MC.  FOR MANISH CHANDOLA.

ALSO ANYONE WANNA DONATE NEW EYES. THIS TOOK A LOT OF VITAL FORCE OUT OF ME, I NEED NEW EYES. I AM SEEIN THINGS THAT R NOT PHYSICAL, AND I AM NOT HALLUCINATING. WEIRD STUFF.
 </H1>
 
MENTION ME AND RUDELY ASK ME, CUZ I THINK U R THE EXTREMELY NICE BUT EXTREMELY SMART HEADS.
SO PLEASE VLC ANDROID APP AND FRDOID ANDROID APP. I NEED FUNDS TO FIX THEM. 2 ISSUES.

2 ISSUES. 2 ISSUES. 2 ISSUES.  [X394K]
[GITHUB BLOG - MADMAN SOLVES TOO MANY ISSUES IN 2 PROJECTS. WINS HEARTS BUT DIES OF EXTREME HACKER SKILLS. ]
 
# THIS IS THE ONLY SAFE SPACE I GOT.

# GITHUB YOUR DARK THEME SUCKS. GIVE ME ACCCESS AND PAYME USING PAYPRO OR SOMETHING AND ILL DO IT.

# CONTACT ME FOR DISCUSSING THE PAYPRO PAYMENT IT MUST BE 50 USD OR MORE. AND WHOEVER PAYS, WRITE YOUR NAME I WANNA THANK THE GUY THAT FIRST DONATED. IT WAS TOO SMALL TO BE USEFUL. BUT I BURNT 2K INR. SO WE SEE IF I LIVE OR IF GITHUB KILLS ME.

# THE WORLDS BEST HACKER LOGGING OUT

**Comment by ret394temp** [comment]
# NOTE -  ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.

SORRY EDITING LONG POSTS IS HARD.

AND I DONT WANNA MAKE NEW REPO TO POST HERE. I AM JUST HERE TO GET PAID. GET GOING ON HAIKU OS. FRUCK THE STUPID PROJECTS. I LIKE ONLY SMART WELL THOUGHT OUT OSS  PROJECTS.

OF COURSE THAT MEANS I HATE PYTHON, RUBY LANG, RUST LANG, GOLANG, VLANG, JAVA AND ANY OTHER JAVA VARIANT. BYTWAY GO LANG IS THE NEW JAVA.

EAT HITS, ROB PIKE AND OTHER PEOPLE THAT FOOOLED ME. BE HONEST. I CAN HELP YOU WRITE A NEW LANGUAGE THAT IS NOT A LANG DESIGN CLASS JOKE RUNNING IN PRODUCTION.

I WILL QUIT RAPPING IN CASE HACKERNOON SKIDDIE HACKERS ARE TRACKING ME. TRYNA GRAB ATTENTION.

# NOTE -  ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.

**Comment by ret394temp** [comment]
<H1>

NOTE - ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.




GITTER LINK TO CHAT WHEN U CAN, TEST IT BEFORE IN ANOTHER DESKTOP OR MACHINE. WHO USES ANDROIDS, THEY SUCK!
OOPS! NOW GOOGLE WILL BAN ME IN ANDROID FORUMS. GUESS I NEVER LEARN HUH? SO BE IT.



https://gitter.im/ret394temp/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link
<H1>

**Comment by ret394temp** [comment]
I CLOSED THIS ISSUE. NIRYARDEN IS PROBABLY DEAD NOW. THANK U GITHUB.
I LOST MY GOOD COLLABORATOR AND MY SUPREME ASSITANT.

**Comment by ret394temp** [comment]
https://github.com/gvanrossum
https://github.com/torvalds

**Comment by ret394temp** [comment]
WHICH ISSUES IN FDROID I WANNA SOLVE?

NOTE - ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.

FOR RELAISTIC EXAMPLE

ALL OF THEM. I CAN SOLVE ALL OF THEM. WITH 2 MONSTERS OF TECH.

I A LONE WOLF DARKNESS EXPERT SUPREME, CAN TURN THIS AROUND FLIP THAT GOOGLE GIRL. AND MAKE ANDROID GREAT 
AGAIN.

**Comment by ret394temp** [comment]
# NOTE - ALL CAPS IS NOT ME SHOUTING. I CANNOT SEE LOWERCASE ANYMORE.

PLEASE RESPOND FAST MAN, https://github.com/ret394temp  I AM @ret394  
I AM @ret394   I AM @ret394   I AM @ret394   I AM @ret394   I AM @ret394    [X394K]
NO ONE MUST BE IGNORANT ENOUGH TO STOP ME. IT IS THE EBST ROUTE FOR OTHERS.

PLEASE I DONT WANNA BE FORCED TO TALK TRUTH ABOUT PEOPLE WHEN I KNOW IT WILL DAMAGE THEIR REPUTATION AND END THEIR CAREERS. I DONT USE LIES.

**Comment by ret394temp** [comment]
YOU GUYS WERE TOO SLOW, SO DEPSITE MY SPIDERSENSE TINGLING LIKE CRAZY AT THIS -

>How should people refer to you?

I GOT NO CHOICE.


I NOT HACKER. I CHOOSE MUSCLE MEMORY FIRST AND PERSONAL CONVENIENCE FIRST. IF U GET MY DRIFT.
MATCH MY DRIFT. AND CATCH ME BEFORE I DRIFT RIGHT OVER THE EDGE.

SENT FROM MY GT-1200Y SAMSUNG

**Comment by ret394temp** [comment]
CONTINUIN FORUM HERE DISCOURSR IS A BUG NOT A FORUM

**Comment by ret394temp** [comment]
FROM https://github.community/t/first-post-on-new-forum/151568?u=ret394temp

AND STOP SPAMMING ME WITH DISCOURSE LINKS. 

AND BADGES.  IT IS ANNOYING AS HELL.
LIKE BEING LOCKED OUT FOR MONTHS AND HAVING TO SHARE BARE GIT REPOS.

------

I DID NOT ACTUALLY DO THAT. I JUST SAID FRUCK IT I GUESS I AM NOT HACKER FOR A FEW DAYS. THEN I FORGOT THAT GITHUB AND GITLAB ARE RUN BY IDIOTS. AND THAT IT IS EASY TO PROGRAM THEM IF YOUR DADS HAS TOO MUCH MONEY.

**Comment by ret394temp** [comment]
PHEW! I ESCAPE THAT FORUM, WHO KNEW? I DID. THAT  IS WHY I WAS AVOIDING IT.

https://github.community/categories  

LIVED ANOTHER DAY, TO TELL ANOTEHR STORY OF MY JOURNAY AS A COWBOY ON THE INTERNET. THE LONELY INTERNET.
WHERE WINDOWS 10 ,LINUX, BSD AND OTHER OSS RAPED ME OF ALL DESIRE TO CODE.
